sklearn/svm/tests/test_svm.py

```diff
@@ -439,6 +439,76 @@ def test_sample_weights():
     assert_array_almost_equal(dual_coef_no_weight, clf.dual_coef_)
 
 
+@pytest.mark.parametrize(
+    "CLF, params",
+    [(svm.SVC,
+      {"err_msg":
```

Comment:
you don't need a dictionary for err_msg. You can pass the string directly and have a `err_msg` parameter

Reply:
Right. Done